packages/yoga/src/Checkbox/web/CheckMark.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { bool, string, func } from 'prop-types';
+import styled from 'styled-components';
+import { hexToRgb } from '@gympass/yoga-common';
+import { HiddenInput } from '../../shared';
+import check from './CheckIcon';
+
+const CheckMarkStyled = styled.div(
+  ({
+    checked,
+    disabled,
+    error,
+    theme: {
+      components: { checkbox },
+    },
+  }) => `
+    position: relative;
+    width: ${checkbox.width}px;
+    height: ${checkbox.height}px;
+    border-radius: ${checkbox.border.radii}px;
+    border-width: ${checkbox.border.width}px;
+    border-style: solid;
```

Comment:
`solid` is the default value of `border-style`. You could omit this value.

Reply:
I have tested and this `border-style: solid` is necessary, isn't default.